packages/components/src/templates/drawer.stories.ts

```diff
@@ -54,29 +54,111 @@ export const Default = {
     </style>
 
     <sd-drawer open placement="end">
```

Comment:
if I close the sd-drawer, I cannot open it again unless I reload the page. 
![image](https://github.com/user-attachments/assets/1f5f8314-94f0-4526-826b-39cdb1232b29)

Is this intended to behave like this? Did you talk about adding a trigger or something to open it up again?

Reply:
Indeed, it’s a bit annoying having to refresh the page to open the drawer. I took the liberty of adding a simple button to open the drawer in case it closes.